charts/kong/values.yaml

```diff
@@ -439,17 +439,19 @@ migrations:
 dblessConfig:
   # Either Kong's configuration is managed from an existing ConfigMap (with Key: kong.yml)
   configMap: ""
+  # Or Kong's configuration is managed from an existing Secret (with Key: kong.yml)
+  secret: ""
   # Or the configuration is passed in full-text below
-  config: |
-    _format_version: "1.1"
-    services:
-      # Example configuration
-      # - name: example.com
-      #   url: http://example.com
-      #   routes:
-      #   - name: example
-      #     paths:
-      #     - "/example"
+  config: "" # |
```

Comment:
note the added commenting: we need this to actually be an empty string if we want strict mutual exclusion logic (unless we want to make users override this to false)